specs/backlog.md

```diff
@@ -0,0 +1,92 @@
+- The swarming of problems by multiple developers.
+
+A common backlog of MOC issues is one way to capture common understandings and enable rapid on-boarding of development resources. 
+
+The team has settled on GitHub Projects and Issues as the tool of choice. It meets the requirements outlined in the first section above and is tightly coupled with our repository of choice, GitHub. 
+
+## Policy
+
+The items in a backlog should be mutable both in their contents and their relative prioritization. Those changes should occur because of actions of the team, or team members, not due to side effects of the underlying tool.
+
+To ensure that level of order and flexibility issues should created in the repository closest to the relevant code and also tracked in the backlog for the development team.  
+
+Github supports this model inherently in their [Kanban model](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-project-boards) and, with a little bit of work can support it for scrum as well.  <--- we will need to validate by usage that the overhead is not onerous.  Also, a discussion of [Kanban or Scrum](https://www.mountaingoatsoftware.com/blog/when-kanban-is-the-better-choice) --->
+
+1. Issues should be written associated with the repository closest to the section of relevant code.  For example a hil/bmi issue would be created in CCI-MOC/hil. If no repository is appropriate they may be added directly to the [mocbacklog](https://github.com/orgs/CCI-MOC/projects/8).   
+2. If you add the issue to the repository,whenever possible add the repository to [Linked Repositories](https://github.com/orgs/CCI-MOC/projects/8/settings/linked_repositories) under the Menu.  This way it will be included in default searches.  
+3. To find and order all cards available across all cci-moc repositories execute this search in the [add card](assets/githubaddcards.png) dialog:
+ 
+    org:CCI-MOC state:open is:issue is:open repo:*
+ 
+### Backlog Setup/Additions
+1. Open [mocbacklog](https://github.com/orgs/CCI-MOC/projects/8)
+2. Click + Add cards on the right side of the project board
+3. In the search field use this search: `org:CCI-MOC is:open is:issue repo:*`,  making sure "Only show results from linked repositories" is *not* checked.
+4. Drag the cards that show in the search results to the BackLog column add them <--- as a team figure out if we need two columns, one for order or "Ready to be worked" --->.
+   It should look something like [this](assets/fillbacklog.png) when you do it. 
+    
+### Scrum Setup
+1. Open [mocbacklog](https://github.com/orgs/CCI-MOC/projects/8) 
+2. Create a new Scrum Project with the name based on starting and end dates - eg. Sprint week 46 and 47 based on number for weeks of year.  
+3. In Sprint Planning meeting move items from Backlog to Sprint Project. 
```

Comment:
Just realized, I haven't said thanks - thanks for reviewing, and the constructive feedback!

did some digging and this needs a rewrite.  Doing it here first then figuring out the best way to capture.  
3. In Sprint Planning Meeting items in [mocbacklog](https://github.com/orgs/CCI-MOC/projects/8) are added to a sprint project board following [these instructions](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/adding-issues-and-pull-requests-to-a-project-board#adding-issues-and-pull-requests-to-a-project-board-from-the-sidebar).